.github/workflows/test_new_highway.yml

```
@@ -7,7 +7,9 @@
 
 name: Update branch with updated submodule on libjxl
 
-on: repository_dispatch
+on:
```

```suggestion
on:
  workflow_dispatch:
```
Let's also add an easy way to trigger it manually..

---

👍